cmd/block.go

```diff
@@ -137,9 +136,8 @@ func blockCmdChain() *cobra.Command {
 			}
 		},
 	}
-	blockLookupCmd.Flags().Bool(headers, false, "option to return list of block headers")
-	blockLookupCmd.Flags().Bool(blocks, true, "option to return list of blocks")
-	blockLookupCmd.Flags().String(blockid, "", "starting block height value")
+	blockLookupCmd.Flags().Bool(header, false, "option to return block header")
```

Comment:
The block lookup command is still using `headers` instead of `header` on line 57 making it not possible to specify header lookup with this command.

Reply:
pushed